### PR TITLE
updated links, phrasing and added translations

### LINKED ISSUE #31: INCF recommendation is gone/moved?
Brought up by @agt24 : url from https://open-brain-consent.readthedocs.io/en/stable/recommendations.html points to http://datasharing.incf.org/ni/Data_Sharing_Language_Recommendations for International Coordination Facility (INCF) Neuroimaging taskforce, and that one is gone now.
@dnkennedy @jbpoline -- do you know where it is now if anywhere?

### PR DESCRIPTION
**Pull request by CPernet**
Closes #31

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -7,40 +7,38 @@ Make open data sharing a no-brainer for ethics committees.
...
-various fields of science.  Neuroimaging is a driving force today
-behind many fields of brain sciences.  Despite possibly terabytes of
-neuroimaging data collected for research daily, just a small fraction
-becomes publicly available. Partially it is because management of
-neuroimaging data requires to confirm to established legal norms,
+The ideology of open and reproducible science makes its ways into various fields
+of science.  Neuroimaging is a driving force today behind many fields of brain
+sciences. Despite possibly terabytes of neuroimaging data collected for research
+daily, just a small fraction becomes publicly available. Partially it is because
+management of neuroimaging data requires to confirm to established legal norms,
```
Comment (yarikoptic) [inline README.rst:14]: note that due to paragraph reformatting it is hard to impossible to see any actual differences

File: docs/source/recommendations.rst [hunk 2]
```diff
@@ -3,16 +3,25 @@
...
+Canada: `NCEHR <http://www.ncehr-cnerh.org/>`_ is a national organization
+mandated to advance the protection and well-being of human participants in
+research and to foster high ethical standards for the conduct of research
+involving humans.
+EU: The General Data Protection Regulation `(GDPR <https://gdpr.eu/>)`_ governs
+how any data (i.e. including research data) should be managed.
+USA: `The Office for Human Research Protections (OHRP <https://www.hhs.gov/ohrp/>)`_
+provides leadership in the protection of the rights, welfare, and wellbeing of
+human subjects involved in research conducted or supported by the U.S.
+Department of Health and Human Services (HHS).
```
Comment (yarikoptic) [inline docs/source/recommendations.rst:20]: make it into an itemized list? otherwise it would all be reformatted into a single paragraph

**Comment by yarikoptic** [comment]
ok, created teams for every language I has spotted: https://github.com/orgs/con/teams/i18n/teams  and invited where I could find some github handles for previous contributors and reviewers.  Invited you to the i18n team itself.  If you have github handles for those who contributed to any of those translations -- could you try to invite into the corresponding team (after accepting my invitation)?

File: README.rst (hunk 1 above)
Comment (yarikoptic) [reply README.rst:14]: I will push this change to also demo our new feature -- RTD test builds in PRs ;)

File: docs/source/recommendations.rst (hunk 2 above)
Comment (yarikoptic) [reply docs/source/recommendations.rst:20]: I will push this change to also demo our new feature -- RTD test builds in PRs ;)

File: README.rst (hunk 1 above)
Comment (CPernet) [reply README.rst:14]: the translations were made mostly by non github people - often MD checking with their data officer (tiny variations here and there to reflect local rules) ;; I'd quite like to move forward as see changes on readthedoc so we can submit the paper (but understand you are cautious)

File: docs/source/recommendations.rst (hunk 2 above)
Comment (CPernet) [reply docs/source/recommendations.rst]: not sure what this implies for rst? just #.

Comment (CPernet) [reply docs/source/recommendations.rst]: yes indeed, thx

File: README.rst (hunk 1 above)
Comment (yarikoptic) [reply README.rst:14]: ok, then -- let's merge but I would still like to get at least #43 (or better with #42) before we "roll it out"

**Pull request merged**